### PR TITLE
Use artifacts for resource name

### PR DESCRIPTION
**Pull request by SinaChavoshi**
Switching to V2 artifact passing format using resource path instead of directly writing to GCS. This PR does not cover updating the metadata, we will do that in a separate follow up PR.

### CODE REVIEW

**Review by sasha-gitg — APPROVED** [review]
Nice work. Thanks Sina!

File: components/google-cloud/google_cloud_components/aiplatform/remote_runner.py [hunk 1]
```diff
@@ -49,56 +45,33 @@ def split_args(kwargs: Dict[str, Any]) -> Tuple[Dict[str, Any], Dict[str, Any]]:
...
     return value
 
 
-def resolve_init_args(key, value, project):
+def resolve_init_args(key, value):
     """Resolves Metadata/InputPath parameters to resource names."""
     if key.endswith('_name'):
-        if value.startswith('gs://'):  # not a resource noun
-            value = read_from_gcs(project, value)
+        if value.startswith('/gcs/'):  # not a resource noun
```
Comment (Ark-kun) [inline components/google-cloud/google_cloud_components/aiplatform/remote_runner.py:73]: Note that this is implementation detail.

File: components/google-cloud/google_cloud_components/aiplatform/remote_runner.py [hunk 2]
```diff
@@ -49,56 +45,33 @@ def split_args(kwargs: Dict[str, Any]) -> Tuple[Dict[str, Any], Dict[str, Any]]:
...
-            value = read_from_gcs(project, value)
+        if value.startswith('/gcs/'):  # not a resource noun:
+            value = read_from_artifact(value)
     return value
 
 
-def resolve_init_args(key, value, project):
+def resolve_init_args(key, value):
     """Resolves Metadata/InputPath parameters to resource names."""
     if key.endswith('_name'):
```
Comment (Ark-kun) [inline components/google-cloud/google_cloud_components/aiplatform/remote_runner.py:72]: I wonder why names come through files, nor values?

**Comment by Ark-kun** [comment]
Thank you, @SinaChavoshi .
This simplifies the code and improves compatibility.

**Pull request merged**